SPECS/cpio/cpio.spec

```diff
@@ -22,8 +24,12 @@ Requires: %{name} = %{version}-%{release}
 %description lang
 These are the additional language files of cpio
 
+# Adjust the fuzz factor for patch command
+%global _default_patch_fuzz 2
```

Comment:
Instead of adding the fuzz, which may potentially introduce invalid application of the patch, we update the patch to match our sources.

Reply:
sure :)